### ISSUE TITLE
SSAO

### ISSUE THREAD

**Issue by junevillarreal** [hzqst/MetaHookSv#489]
I'd like to report maybe a bug or something? SSAO seems not working in all version of CS1.6. I tried in 3 different client, 2 clients are heavily modded (CSPB & Hellstrike), and the other is CS1.6 client 8684. SSAO only works in CSPB, but not in Hellstrike or 1.6 build 8684.

This is what SSAO looks like in CSPB
![ssao2](https://github.com/user-attachments/assets/ec00dadc-c6c4-47cd-9f07-e35dbe786248)
![ssao1](https://github.com/user-attachments/assets/c1a7bf9d-4775-4f91-b845-127b7be732ae)
![ssao3](https://github.com/user-attachments/assets/0c4656f6-8a1d-4f36-af28-fa797febc699)

No SSAO in Hellstrike 
![ssao6](https://github.com/user-attachments/assets/bf719616-10f6-4a8b-b7d2-be77040cd37b)

No SSAO in CS 1.6 build 8684
![ssao4](https://github.com/user-attachments/assets/669cdbc7-0e77-4c40-a961-5a3923ec75e0)
![ssao5](https://github.com/user-attachments/assets/e0261076-d016-4879-a817-b8801df49615)

**Comment by hzqst** [comment]
is this the same issue ? https://github.com/hzqst/MetaHookSv/issues/464

**Comment by junevillarreal** [comment]
> is this the same issue ? #464

yeah i guess it is. but the weird thing is in CSPB the r_vertical_vof is set to 0, and the viewmodel_fov is 0 too, but the SSAO works.

**Comment by hzqst** [comment]
> > is this the same issue ? #464
> 
> yeah i guess it is. but the weird thing is in CSPB the r_vertical_vof is set to 0, and the viewmodel_fov is 0 too, but the SSAO works.

it depends on actual fov sent from server (maybe amxx?) then.

**Comment by hzqst** [comment]
Fixed in https://github.com/hzqst/MetaHookSv/releases/tag/v20250307b
